core/cap-0035.md

```diff
@@ -226,6 +383,42 @@ Possible return values for the `ClawbackOp` are:
 - `CLAWBACK_UNDERFUNDED` if the `from` account does not have sufficient
   available balance of `asset` after accounting for selling liabilities.
 
+#### Create Claimable Balance Operation
+
+This proposal introduces a new failure case in the execution of the
+`CreateClaimableBalanceOp`. If the source account of the operation has a
+trustline to the asset with the `CLAWBACK_ENABLED_FLAG` flag set, the
+operation will fail with `CREATE_CLAIMABLE_BALANCE_CLAWBACK_REQUIRED` if one
+of the `claimants` `destination` is not the issuer account of the asset, or
+if the issuer account is a claimant and the `predicate` is not

```

Comment:
This doesn't cover quite enough cases. What about the case where the issuer should be able to clawback or do a normal claim?

Reply:
I've removed the predicate completely now.